src/Component/ProcessRunner/ProcessRunner.php

```diff
@@ -54,6 +55,8 @@ public function run(Host $host, string $command, array $config = []): string
             $terminalOutput($type, $buffer);
         };
 
+        $command = $this->replacePlaceholders($command, $config['vars']);
```

Comment:
Awesome PR 👏
But there is support in `run()`?

Reply:
Crap, it's missing. Will follow up with another commit.

I've tried to add unit tests, however it seems that `on(localhost(), function () { run() })` is not working in unit tests at the moment.

Reply:
Unfortunately, I don't know how to add unit tests for that, as the `on()` will not work, and setting localhost in the context results in the logic to use the `ProcessRunner` instead of `Ssh/Client`

The implementation has been added in the c8657c425c1c1db8ecd12265abeac9e78912836e